tests/ui/const-generics/generic_const_exprs/post-analysis-user-facing-param-env.rs

```diff
@@ -0,0 +1,18 @@
+// Regression test for #133271.
```

Comment:
does that fix the issue? asking since I don't see anything that would autoclose it :thinking:

Reply:
true :3 thx